rllab/envs/dm_control_env.py

```diff
@@ -0,0 +1,82 @@
+import pygame
+import numpy as np
+
+from dm_control import suite
+from dm_control.rl.environment import StepType
+
+from rllab.envs.base import Env, Step
+from rllab.envs.dm_control_view import DmControlViewer
+from rllab.core.serializable import Serializable
+from rllab.spaces.box import Box
+'''
+This environment will use dm_control toolkit(https://arxiv.org/pdf/1801.00690.pdf) 
+to train and simulate your models.
+'''
+
+
+def _flat_shape(observation):
```

Comment:
You may find these useful https://github.com/ryanjulian/rllab/blob/integration/rllab/misc/tensor_utils.py

If none of those utils help, please add some new ones so your functions can be reused!

Reply:
https://github.com/deepmind/dm_control/blob/master/dm_control/rl/control.py#L369